npg_qc_viewer/root/static/scripts/qcoutcomes/qc_outcomes_view.js

```diff
@@ -34,6 +34,38 @@ define([
 ) {
   var ID_PREFIX = 'rpt_key:';
 
+  /*
+   *This function assigns an image in the .lane column to signal the outcome of
+   *the usability flag from a certain tag.
+   *
+   *It requires a DOM object (obj) and a boolean flag (usability) displaying wether
+   *the usability of the tag lane is switched on or off. It also require the presence of the
+   *src images: bigTickBlue.png and bigCrossRed.png at the right path "/static/images/"
+   *
+   * Example:
+   *
+   *    $('.lane').each(function (i, obj) {
+   *     mqc_outcomes.usabilityDisplaySwitch(obj,1);
+   *   });
+   *
+   */
+  var usabilityDisplaySwitch = function (obj, usability) {
+    if (typeof obj === 'undefined') {
+      throw new TypeError('Parameter obj must be defined');
+    }
+    if (typeof usability !== 'boolean') {
+      throw new TypeError('Usability should be boolean');
+    }
+    var o = $(obj);
+    var img;
+    if(usability){
+      img = $('<div class="utility_pass">&#10003;</div>');
```

Comment:
Maybe explain in comment what &#10003; and &#10008; stand for since they are strange constants
// hex symbols
// check &#10003;
// cross &#10008;